hugo/layouts/partials/header.html

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-expand-md navbar-dark fixed-top bg-gradient">
+      <a class="navbar-brand" href="/"><img src="logo.svg" /></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
```

Comment:
Great with the ability to port the rest of the pages! Can we hide them in the initial release?

Reply:
Sure we can, will look into it soon.